test/files/stream.iced

```diff
@@ -0,0 +1,52 @@
+crypto = require('crypto')
+to_buf = require('../../src/stream-to-buffer.iced')
+stream = require('../../src/chunk-stream.iced')
+
+stream_random_data = (strm, len, cb) ->
+  written = 0
+  expected_results = []
+  while written < len
+    # generate random length
+    await crypto.randomBytes(1, defer(err, index))
```

Comment:
How hard would it be to have these tests take a single random seed, log it, and then have everything be deterministic after that? The worry with making it actually random is that you'll get a failure in the logs, and then you won't be able to reproduce it, and it'll be ambiguous whether you can't reproduce it because your environment is different or because the input was just very unlikely.